src/core/api/api_error.pl

```diff
@@ -201,7 +201,7 @@
     ->  Opening_Msg = "Remote authentication failed"
     ;   Opening_Msg = "Remote connection failed"
     ),
-
+    % Needs more options....
```

Comment:
What is this comment about?

Reply:
Another bug :D This needs to capture more cases.